src/libutil/terminal.hh

```diff
@@ -21,16 +21,12 @@ std::string filterANSIEscapes(std::string_view s,
     bool filterAll = false,
     unsigned int width = std::numeric_limits<unsigned int>::max());
 
-#ifndef _WIN32
-
 /**
  * Recalculate the window size, updating a global variable. Used in the
  * `SIGWINCH` signal handler.
```

Comment:
I guess this comment is outdated then?

Reply:
Sorry, I'm not sure what you mean. Are you referring to the part of the comment that says it's used in the SIGWINCH handler?

Reply:
Never mind, I think I was wrong.